<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>333</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/icons/agt_softwareD-64.png</normaloff>:/icons/agt_softwareD-64.png</iconset>
  </property>
  <property name="locale">
   <locale language="Dutch" country="Belgium"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="g_contactgegevens">
     <property name="title">
      <string>Contact data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="l_naam">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="7">
       <widget class="QLineEdit" name="m_naam"/>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLabel" name="l_straat">
        <property name="text">
         <string>Straat (+ huisnummer / bus)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" colspan="4">
       <widget class="QLineEdit" name="m_straat"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l_postcode">
        <property name="text">
         <string>Postal code</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QSpinBox" name="m_postcode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>1000</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="QLabel" name="l_gemeente">
        <property name="text">
         <string>City</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5" colspan="3">
       <widget class="QLineEdit" name="m_gemeente"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="l_telefoon">
        <property name="text">
         <string>Telephone</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="5">
       <widget class="QLineEdit" name="m_telefoon"/>
      </item>
      <item row="3" column="6">
       <widget class="QLabel" name="l_gsm">
        <property name="text">
         <string>GSM</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QLineEdit" name="m_gsm"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="l_email">
        <property name="text">
         <string>E-mail</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="7">
       <widget class="QLineEdit" name="m_email"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="l_onderschrift">
        <property name="text">
         <string>Onderschrift</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="6">
       <widget class="QLineEdit" name="m_onderschrift"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="g_instellingen">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="l_riziv">
        <property name="text">
         <string>Riziv</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="m_riziv"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="l_btwpercentage">
        <property name="text">
         <string>VAT percentage</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="m_btwpercentage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="l_btwnummer">
        <property name="text">
         <string>BTW nummer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="m_btwnummer"/>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="l_rekeningnummer">
        <property name="text">
         <string>Rekeningnummer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="m_rekeningnummer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_annuleren">
       <property name="text">
        <string>Annuleren</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
